app/report/page.tsx

```diff
@@ -0,0 +1,50 @@
+'use client';
+
+import { Cards, Summary } from '@/components/Report';
+import TopBar from '@/components/TopBar';
+import { Flex, Text, VStack } from '@chakra-ui/react';
+import type { Metadata } from 'next';
+
+const navy = '#1E1E1E';
+
+export const metadata: Metadata = {
+  title: 'Report',
+  description: 'Your summarised health report',
+  openGraph: {
+    title: 'Upload health report',
+    description: 'Upload your health report for analysis',
+    url: 'https://mediclear.vercel.app',
+    siteName: 'MediClear',
+    images: [
+      {
+        url: 'https://mediclear.vercel.app',
+        width: 800,
+        height: 600
+      }
+    ],
+    locale: 'en-US',
+    type: 'website'
+  }
+};
+
+const Report = () => {
+  return (
+    <Flex w='100%' h='100vh' flex={1} direction='column'>
```

Comment:
Why are u setting the height to 100vh? This page is meant to be one page and needs to be scrollable.

Reply:
Thanks for catching this!